api/controller.go

```diff
@@ -59,12 +59,11 @@ type Dependencies struct {
 	Cataloger       catalog.Cataloger
 	Auth            auth.Service
 	BlockAdapter    block.Adapter
-	Stats           stats.Collector
```

Comment:
[![stats or collector?  why can't we have both?!](https://user-images.githubusercontent.com/7802932/106383538-5fb4da00-63cf-11eb-88f8-d028d11146b8.gif)](https://imgflip.com/gif/4w2ztr)

Reply:
OMG gifs werk in github.  The internet is _so_ over.

Reply:
wonder how much it is easy to see that we are passing the same service twice :)